docs/source/admin/traffic_ops/using.rst

```diff
@@ -441,7 +441,7 @@ The fields in the Delivery Service view are:
 |                                            |                                                                                                                                                                        |
 |                                            | **Note:** you will not be able to save a Regex Remap Expression if you have Query String Handling set to drop query strings at the edge. See :ref:`regex-remap`.       |
 +--------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| Cache URL expression                       | Deprecated.                                                                                                                                                            |
+| Cache URL expression                       | Deprecated in Traffic Control 3.x and subject to removal in Traffic Control 4.x or later.                                                                                                                                                            |
```

Comment:
This leaves the table malformed. The table borders and column/row delimiters need to remain intact for it to compile. Should just be a whitespace change.